src/controllers/recordController.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const recordService = require("../services/recordService");
+const { throwError } = require("../utils/throwError");
+const { readRecordService, createRecordService } = recordService;
+
+const readRecord = async (req, res) => {
+  try {
+    const { id } = req.params;
+    const readRecordService = await recordService.readRecordService(Number(id));
+    if (readRecordService === undefined) {
+    return res.status(400).json("NO_USER");
+    }
+    res.status(200).json(readRecordService);
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const createRecord = async (req, res) => {
+  try {
+    const addRecord = req.body;
+    const {
+      userId,
+      waterContent,
+      workoutTime,
+      currentWeight,
+      muscleMass,
+      bodyFat,
+      maxHeartrate,
+    } = addRecord;
```

Comment:
```suggestion
    const recordData = req.body;
    const {
      userId,
      waterContent,
      workoutTime,
      currentWeight,
      muscleMass,
      bodyFat,
      maxHeartrate,
    } = recordData;
```
- 변수는 명사형으로 지정하시는 것이 JS 컨벤션을 지키기에 보다 좋습니다 :)

Reply:
반영했습니다.